src/main/java/org/influxdb/dto/Point.java

```diff
@@ -336,15 +326,18 @@ public String lineProtocol() {
      * @return the String without newLine
      */
     public String lineProtocol(final TimeUnit precision) {
-      final StringBuilder sb = CACHED_STRINGBUILDERS
-              .get()
-              .computeIfAbsent(this.measurement, MeasurementStringBuilder::new)
-              .resetForUse();
+      final StringBuilder sb = CACHED_STRINGBUILDERS.get();
+      sb.setLength(0);
```

Comment:
Just to be kind to the next developer who wonder why we do this here, I would add a comment explaining our intention.